### PR TITLE
Add threshold to tag grants

### PR DESCRIPTION
**Pull request by nsorros**
https://trello.com/c/thI1pvAF

Tagging sample of grants for quality control by Kirstin. This PR adds the ability to tag with different thresholds, the intention is to tag with thresholds 0.3, 0.5, 0.7, 0.9 and kick off discussion about what is the appropriate default.

### CODE REVIEW

**Comment by nsorros** [comment]
[Tag grants with new ensemble model for quality check from Kirstin. Use four different threshold to kick discussion about better default](https://trello.com/c/thI1pvAF/1036-tag-grants-with-new-ensemble-model-for-quality-check-from-kirstin-use-four-different-threshold-to-kick-discussion-about-better-d)

File: grants_tagger/tag_grants.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def yield_grants(grants_path):
         for grant in csv_reader:
             yield grant
 
-def tag_grants(grant):
+def tag_grants(grant, threshold):
```
Comment (lizgzil) [inline grants_tagger/tag_grants.py:18]: is this going to break anything by not putting in a default value? (i.e. is tag_grants being used anywhere else without the threshold argument)

Comment (nsorros) [reply grants_tagger/tag_grants.py:18]: not but good point, for backwards compatibility i should add a default

**Comment by nsorros** [comment]
> Maybe you can tag also with Liz's formula (F1_Score/2)

this is if you want to maximise f1

**Comment by nsorros** [comment]
Adding tests is not straightforward as explained in #19

**Pull request merged**